README.md

```diff
@@ -83,6 +84,42 @@ Validate the activated git hooks by running
 
     poetry run autohooks check
 
+## Configuration
+
+The configuration of notus-scanner can be done by providing a TOML config file.
+Per default notus-scanner tries to load the settings from config files in the
+following order: `~/.config/notus-scanner.toml`, `/etc/gvm/notus-scanner.toml`.
+
+Alternatively the location of the to be loaded config file can be set via the
+`-c`/`--config` command line argument. Setting a config file via command line
+will ignore the default config files.
+
+The settings are read from a `[notus-scanner]` [section](https://toml.io/en/v1.0.0#table).
+
+Example config file:
+```toml
+[notus-scanner]
+mqtt-broker-address = "1.2.3.4"
+mqtt-broker-port = "1234"
+advisories-directory = "/tmp/notus-scanner/advisories"
+pid-file = "/tmp/notus-scanner.pid"
+log-file = "/tmp/notus-scanner.log"
+log-level = "DEBUG"
+```
+
+Each setting can be overridden via an environment variable or command line
+argument.
+
+|Config|Environment|Default|Description|
+|------|-----------|-------|-----------|
+|advisories-directory|NOTUS_SCANNER_ADVISORIES_DIRECTORY|/var/lib/openvas/plugins/notus/products|Directory for loading JSON advisories|
```

Comment:
Shouldn't this option be renamed to products? it is a bit confusing, since inside the `<feed>/notus/` directory there are two folders `advisories` and `products`. While the option is named advisories, it expects the products directory path.

Reply:
i've renamed the argument and variables to products directory. initially it was intended to point to the directory where both types of advisory data are contained in the subfolders advisories and products. With loading the advisory meta data in ospd-openvas it makes sense to be more precise here.